src/Query.php

```diff
@@ -486,4 +486,19 @@ public function compileQuery()
 
         return $this->_elasticQuery;
     }
+
+    public function aliasField($field)
+    {
+        return [$field];
+    }
+
+    public function alias()
+    {
+        return $this->_repository->alias();
+    }
+
+    public function find($type, $options)
+    {
+        return $this->_repository->find($type, $options);
+    }
```

Comment:
This will need a way of passing the original query in so that conditions from the first finder are not lost in the second one.